src/evo/deterministicmns.cpp

```diff
@@ -417,9 +421,13 @@ void CDeterministicMNList::RemoveMN(const uint256& proTxHash)
     auto dmn = GetMN(proTxHash);
     assert(dmn != nullptr);
     DeleteUniqueProperty(dmn, dmn->collateralOutpoint);
-    DeleteUniqueProperty(dmn, dmn->pdmnState->addr);
+    if (dmn, dmn->pdmnState->addr != CService()) {
```

Comment:
```suggestion
    if (dmn->pdmnState->addr != CService()) {
```

Reply:
oh my :see_no_evil: ... that stuff like this even compiles is miracle only observed in C++...

Reply:
This is actually a legit construction, just doesn't make sense to use it here :)